test/unit/presenters/publishing_api_presenters/take_part_test.rb

```diff
@@ -0,0 +1,51 @@
+require 'test_helper'
+
+class PublishingApiPresenters::TakePartTest < ActiveSupport::TestCase
+  def present(record)
+    PublishingApiPresenters::TakePart.new(record).as_json
+  end
+
+  test "case study presentation includes the correct values" do
+    # TODO move setting content_id to model
```

Comment:
Only that they'll all need to do it so it made sense to leave it in the module as it is required for the Publishing API. It could go in its own module I suppose.